src/style/variables.sass

```
@@ -7,32 +7,32 @@
     --color-secondary-dark: hsl(13, 100%, 15%)
     --color-secondary: hsl(13, 100%, 30%)
 
-    --bg-color-primary: hsl(0, 0%, 0%)
-    --bg-color-secondary: hsl(0, 0%, 10%)
+    --bg-color-primary: #1D1D1D
+    --bg-color-secondary: #131313
     --bg-color-tertiary: hsl(0, 0%, 20%)
     --bg-color-quaternary: hsl(0, 0%, 30%)
 
     --text-color-primary: hsl(0, 0%, 100%)
     --text-color-secondary: hsl(0, 0%, 80%)
 
     // Component colors
-    --bg-color-header: hsl(0, 0%, 0%)
-
+    --bg-header-top: #192619
+    --bg-header-bottom: #131c13
     --text-color-header: hsl(0, 0%, 100%)
     --text-color-side-navigation: hsl(0, 0%, 100%)
-
+    --code-default-color: #efc622
```

- `code` is not an component, should be relocate to global colors.
- The name `code-default-color` should match the other variables naming.
```suggestion
    --code-color: #efc622
```